docs/src/main/docs/cdi-extensions.adoc

```diff
@@ -0,0 +1,217 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Helidon CDI extensions
+:description: Helidon CDI extensions guide
+:keywords: helidon, guide, CDI
+
+* <<Overview>>
+* <<Installing a CDI extension>>
+* <<Injecting an object using a CDI extension>>
+
+== Overview
+
+Helidon provides https://docs.jboss.org/cdi/spec/2.0/cdi-spec.html#spi[portable extensions] that you can use to inject the following objects into your Helidon MicroProfile applications:
+
+* HikariCP data sources
+* Jedis clients
+* Oracle Cloud Infrastructure Object Storage clients
+
```

Comment:
Add a line or two describing each of these services.

Reply:
@ljnelson - Is it worth mentioning why we picked these 3 services, and possibly mention that we’ll provide extensions for other services in the future, without naming any specifically of course?

Reply:
For each service, I've added a link to the appropriate site.